data/reference_library_documents_from_airtable.csv

```diff
@@ -97,4 +97,4 @@ information among the relevant stakeholders.""",Guideline,International Atomic E
 Zhanat Carr,2017 ITB guideline-web.pdf (https://dl.airtable.com/.attachments/e32d55f8acac732c9c4e17f3260c2477/fea976f9/2017ITBguideline-web.pdf),"Iodine Thyroid Blocking: Guidelines for use in planning for and responding
 to radiological and nuclear emergencies",Iodine thyroid blocking: guidelines for use in planning for and responding to radiological and nuclear emergencies,Radiation Emergencies,18.1 Document and disseminate best practices of test results and reviews.,Guideline,World Health Organization,2017,checked,,All,,2017 ITB guideline-web.png (https://dl.airtable.com/.attachments/b305589f06837a82f58468ea2b853ee1/11456c54/2017ITBguideline-web.png)
 Zhanat CARR,2019 REMPAN directory_final-full.pdf (https://dl.airtable.com/.attachments/a8383c5a2456be9d781b9ba4d2df23da/a1af41cb/2019REMPANdirectory_final-full.pdf),Radiation Emergency Medical Preparedness and Assistance Network Directory,The directory of the members of WHO REMPAN - Radiation Emergency Medical Preparedness and Response Network ,Radiation Emergencies,"""17.1 Establish links with key international chemical/toxicology networks51 for support in the management of chemical events and
-poisonings."",""18.1 Designate or secure access to laboratory capacity to monitor, identify and assess exposure."",18.1 Identify health facilities that can manage patients in the event of a radiation emergency.",Tool,World Health Organization,2019,checked,,All,,2019 REMPAN directory_final-full.png (https://dl.airtable.com/.attachments/ec1bc2d28f9e5f03aa295347a32a628f/84b9d94d/2019REMPANdirectory_final-full.png)
\ No newline at end of file
+poisonings."",""18.1 Designate or secure access to laboratory capacity to monitor, identify and assess exposure."",18.1 Identify health facilities that can manage patients in the event of a radiation emergency.",Tool,World Health Organization,2019,checked,,All,,2019 REMPAN directory_final-full.png (https://dl.airtable.com/.attachments/ec1bc2d28f9e5f03aa295347a32a628f/84b9d94d/2019REMPANdirectory_final-full.png)
```

Comment:
NIT: Ok, I'm guessing that your editor/plugin(s) are re-formatting code for you. But this is not a useful change, its just adding a newline, is that correct? Do we really want to be committing edits to a file, and thereby changing git history, for the sake of whomever edited a file last? I don't think its a good practice. Or, as an alternative, perhaps make a big commit whose purpose is re-formatting code? This is probably a individual/team preference, but in general I think it good to keep code changes related to the purpose of the story at hand. What do you think?

Reply:
yes my editor formatted this. fwiw newline at end of a file is considered to be a useful practice in general: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
I don't have a preference, but I think its ok to re-format files that we touch (Boy Scout principle?). happy to revert though if you want to it all in a big commit later on.